.wp-env.json

```diff
@@ -8,10 +8,7 @@
 				"wp-content/plugins/gutenberg": ".",
 				"wp-content/mu-plugins": "./packages/e2e-tests/mu-plugins",
 				"wp-content/plugins/gutenberg-test-plugins": "./packages/e2e-tests/plugins",
-				"wp-content/themes/gutenberg-test-themes": "./test/gutenberg-test-themes",
-				"wp-content/themes/gutenberg-test-themes/twentytwentyone": "https://downloads.wordpress.org/theme/twentytwentyone.2.1.zip",
-				"wp-content/themes/gutenberg-test-themes/twentytwentythree": "https://downloads.wordpress.org/theme/twentytwentythree.1.3.zip",
-				"wp-content/themes/gutenberg-test-themes/twentytwentyfour": "https://downloads.wordpress.org/theme/twentytwentyfour.1.0.zip"
```

Comment:
I think these themes are actually used in performance tests no?

Reply:
I think if we remove these, the performance tests in CI might still run but locally, you'll have issues running these. cc @oandregal

Reply:
@youknowriad but when I see that under the test folder all three theme folders are empty even before this modification also 🤔

Reply:
They're being downloaded in the test environment, and they're used here:

https://github.com/WordPress/gutenberg/blob/57b8b613d0507fd2c1ed0b441db09f76a43d579f/bin/plugin/commands/performance.js#L422-L430

Reply:
@youknowriad I run a performance test locally and it's working fine. 
`npm run test:performance`

Reply:
This was added in #50905, but the purpose seems to be to keep theme versions consistent across performance tests.

Reply:
I guess we should update the performance test to run on the latest version to avoid duplicate theme installation.